osquery/main/windows/daemon.cpp

```diff
@@ -54,7 +66,124 @@ void DebugPrintf(const char *fmt, ...) {
   va_end(vl);
 }
 
-void UpdateServiceStatus(DWORD controls, DWORD state, DWORD exit_code, DWORD checkpoint) {
+/// Install osqueryd as a service given the path to the binary
+Status installOsqueryService(const std::string& binPath) {
```

Comment:
Also `installOsqueryService` in the `osquery` namespace is a bit redundant, any reason not to prefer `installService`?